hathor/simulator/tx_generator.py

```diff
@@ -70,7 +73,8 @@ def schedule_next_transaction(self):
             self.manager.propagate_tx(self.tx, fails_silently=False)
             self.tx = None
 
-        dt = random.expovariate(self.rate)
+        dt = self.rng.exponential(1 / self.rate)
```

Comment:
Maybe create a `self.mean = 1 / self.rate` and use it here.

Reply:
Refactored out since `rng` now has `expovariate`.